locations/linked_data_parser.py

```diff
@@ -215,3 +217,14 @@ def parse_enhanced_hotel(ld: dict, item: Feature):
                 item["extras"]["stars"] = stars
             elif isinstance(stars, dict):
                 item["extras"]["stars"] = stars.get("ratingValue")
+
+    @staticmethod
+    def parse_same_as(ld: dict, item: Feature):
+        if same_as := LinkedDataParser.get_clean(ld, "sameAs"):
+            if isinstance(same_as, str):
+                same_as = [same_as]
+            for link in same_as:
+                if "facebook.com" in link:
+                    add_social_media(item, "facebook", link)
```

Comment:
For keys such as `facebook` and `tripadvisor` used with `add_social_media`, could these keys be mapped out in a `SocialMedia` (or similar name) Enum?

e.g. before:
`add_social_media(item, "tripadvisor", link)`

after:
`add_social_media(item, SocialMedia.TRIP_ADVISOR, link)`

Reply:
I have no objection, but I would request that anything using it takes both the enum or the string. Like how category stuff can take an enum or dict.